src/phases/mystery-encounter-phases.ts

```diff
@@ -165,6 +166,28 @@ export class MysteryEncounterOptionSelectedPhase extends Phase {
   }
 }
 
+/**
+ * Runs at the beginning of an Encounter's battle
+ * Will cleanup any residual flinches, Endure, etc. that are left over from startOfBattleEffects
+ * See [TurnEndPhase](../phases.ts) for more details
+ */
+export class MysteryEncounterBattleStartCleanupPhase extends Phase {
+  constructor(scene: BattleScene) {
+    super(scene);
+  }
+
+  start() {
```

Comment:
public method => jsdoc

Reply:
jsdoc is on the class, start is consistent to all phases